contrib/hyperscan/matching/input_matchers/test/hyperscan_speed_test.cc

```diff
@@ -38,14 +46,14 @@ static void BM_Hyperscan(benchmark::State& state) {
   hs_database_t* database{};
   hs_scratch_t* scratch{};
   hs_compile_error_t* compile_err;
-  RELEASE_ASSERT(hs_compile_multi(ClusterRePatterns.data(), nullptr, nullptr,
-                                  ClusterRePatterns.size(), HS_MODE_BLOCK, nullptr, &database,
+  RELEASE_ASSERT(hs_compile_multi(clusterRePatterns().data(), nullptr, nullptr,
```

Comment:
make a temp for return value of ClusterRePatterns() rather than repeating its call 3x

Reply:
Thanks, fixed.